js/_wiki.ts

```diff
@@ -6,14 +6,13 @@ export function highlightCodeBlocks(selector: string){
     const baseExtensions = [...extensions.base, EditorState.readOnly.of(true)];
 
     for (const code of $$(selector)) {
-        let lang = code.className.slice('language-'.length).toLowerCase();
-
-        // FIXME Is there a better way to do this mapping?
-        if (lang == 'asm') lang = 'assembly';
-        if (lang == 'c++') lang = 'cpp';
-        if (lang == 'cs' ) lang = 'c-sharp';
-        if (lang == 'fs' ) lang = 'f-sharp';
-        if (lang == 'ijs') lang = 'j';
+        const lang = {
+            'asm' : 'assembly',
+            'c++' : 'cpp',
+            'cs'  : 'c-sharp',
+            'fs'  : 'f-sharp',
+            'ijs' : 'j',
+        }[code.className.slice('language-'.length).toLowerCase()] || '';
```

Comment:
I think that fails to set lang for the ones not in the set.

Reply:
Yeah, it might. Guess I'll have to `make dev` after all...

👀

Reply:
So I think you misunderstand what I mean by the FIXME, my issue isn't the if/else or hash or switch stmt, my issue is the actual need to map these. Your fix doesn't fix the fixme.

Reply:
Suggestion:

```ts
let lang = code.className.slice('language-'.length).toLowerCase();

lang = {
    'asm' : 'assembly',
    'c++' : 'cpp',
    'cs'  : 'c-sharp',
    'fs'  : 'f-sharp',
    'ijs' : 'j',
}[lang] ?? lang
```

Reply:
Oooh, I see. So you just want to _eliminate_ those reassignments. Forgive me, but that comment is rather multi-interpretable.